01week/bio/index.html

```diff
@@ -0,0 +1,74 @@
+    <link rel="stylesheet" href="css/css-reset.css"/>
+    <!--This is my css file-->
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+
+
+</head>
+
+<body>
+    <main>
+        <header>
+            <img src="https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAkgAAAAJGYyMTlhNWJlLTQ1MjctNDYyZS1hYTNkLWZhNTE3YjA1OGMzOA.jpg" alt="Ethan Crowley" />
+            <h1>Ethan Crowley</h1>
+            <h3>Developer</h3>
+        </header>
+        <section>
+            <h3>About me</h3>
+            <p>I grew up in a small west Texas farming town called Crosbyton (about 40 miles east of Lubbock). After high school graduation I moved to Lubbock, but never really felt at home there. After the birth of my son in 2008 it was decided that I would make my way to Austin. I attended Texas State Technical college in Waco and earned my associates which landed me a job at Samsung Austin Semiconductor. For a while life was good but I soon realized that I was not in the profession that I wanted to be in. I thought long and hard about what I enjoyed throughout school and I remembered enjoying the class taught on HTML. I then decided to challenge myself by enrolling in Austin Coding Academy and start a new life.   </p>
+
+
+        </section>
+        <section>
+            <h3>Why I am learning to code</h3>
+            <p>Since I took a simple HTML class in high school I've always wanted to code. Life happened pretty fast after high school, and I ended up with a decent job, it just wasn't what I wanted to do. I want to code because I believe it is the future, and I want to constantly be challenged. </p>
+        </section>
+        <section>
+
+            <section id="shows">
+            <h3>My Favorite TV Shows</h3>
+              <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRr7194h_M4sk1lCYoz25kkGAE-31zOgOoqyXqRQaY5XCshYzra"  alt="Rick and Morty"  />
+              <img src="https://images-na.ssl-images-amazon.com/images/M/MV5BMjEyMDU2MTI1OF5BMl5BanBnXkFtZTgwMjU5NjMxOTE@._V1_SY1000_CR0,0,674,1000_AL_.jpg" width="186 px" height="271 px"  alt="Stranger Things" />
+              <img src="https://images-na.ssl-images-amazon.com/images/M/MV5BMTg2ODI2NTUwN15BMl5BanBnXkFtZTgwMTMwMzU0MjE@._V1_SY1000_CR0,0,913,1000_AL_.jpg" width ="186 px" height="271 px"  alt="Downton Abbey" />
+              <ul>
+                <li>Rick and Morty </li>
+                <li>Stranger Things </li>
+                <li>Downton Abbey </li>
+              </ul>
+            </section>
+
+        <section id="bands">
+            <h4>My Favorite Bands</h4>
+            <img src="http://s3-eu-west-1.amazonaws.com/radioheadassets/deadairspace/img/140/0/090e8da8.jpg" width="200 px" height="200 px" alt="Radiohead"  />
+            <img src="https://lcdsoundsystem.com/wp-content/uploads/2016/05/13130873_10154745390199338_8932651528954991877_o-1024x1024.jpg" width="200 px" width="200px" alt="LCD Soundsystem "  />
```

Comment:
Hey Ethan! I noticed on this line you have a double "width" attribute. Eliminating one might help get you a green check!